docs/source/vector_ar.rst

```diff
@@ -257,9 +257,17 @@ F-test.
 Normality
 ~~~~~~~~~
 
+As pointed out in the beginning of this document, the white noise component :math:`u_t` is assumed to be normally distributed. To test whether this assumption is acceptable or has to be rejected, :class:`VARResults` offers the `test_normality` method.
```

Comment:
`is assumed to be normally distributed`  is not completely accurate for the estimator.
We get consistent parameter estimates even if residuals are not normal, similar to OLS.

I'm not sure whether normality is assumed anywhere in the code/model itself. So maybe just add a note that normality assumption is not necessary to get consistent parameter estimates.